src/pages/groupComparison/GroupComparisonMutationMapperWrapper.tsx

```diff
@@ -151,18 +151,21 @@ export default class GroupComparisonMutationMapperWrapper extends React.Componen
             this.props.store.profiledPatientCounts,
         ],
         render: () => {
-            const mutationMapperStore = this.mutationMapperToolStore.getMutationMapperStore(
-                this.props.store.activeMutationMapperGene!.hugoGeneSymbol
-            );
+            let mutationMapperStore;
+            if (this.props.store.activeMutationMapperGene) {
+                mutationMapperStore = this.mutationMapperToolStore.getMutationMapperStore(
+                    this.props.store.activeMutationMapperGene.hugoGeneSymbol
+                );
+            }
             if (mutationMapperStore) {
                 return (
                     <div
                         data-test="ComparisonPageMutationsTabPlot"
                         style={{ marginTop: 20 }}
                     >
                         <h3>
-                            {this.props.store.activeMutationMapperGene
-                                ?.hugoGeneSymbol +
+                            {this.props.store.activeMutationMapperGene!
```

Comment:
maybe better to combine the condition at 160 with the condition at 155.  then you wouldn't have to do this assertion.